python-backend/app/api/mines/namespace/mines.py

```diff
@@ -49,3 +51,12 @@
 api.add_resource(
     VarianceUploadedDocumentsResource,
     '/<string:mine_guid>/variances/<string:variance_id>/documents/<string:mine_document_guid>')
+
+api.add_resource(MineIncidentListResource, '/<string:mine_guid>/incidents')
+
+api.add_resource(MineIncidentResource, '/<string:mine_guid>/incidents/<string:mine_incident_guid>',
+                 '/incidents/<string:mine_incident_guid>')
```

Comment:
Two endpoints to one resource? That's a pattern we're avoiding now, no? What's the reason for implementing it this way?

Reply:
You are right, meant to remove that, thanks.